### DISCUSSION TITLE
Liked Lists from Trakt

### DISCUSSION THREAD

**Discussion by StrangerJona** [Taxel/PlexTraktSync#1135] [Q&A] [answered]
I've been trying to rack my brain and search for some answers, but coming up short.
Is it at all possible for PTS to import a liked list or similar from Trakt to Plex, my wife is an avid Chicago franchise fan, and is keen to watch them in chronological order, I've found a list. But cannot for the life of me work out how to import it as a playlist to Plex, at least not in 2022. Any advice would be wonderful.

**Comment by simonc56** [comment, marked as answer]
Readme states:

> Liked lists in Trakt are downloaded and all movies in Plex belonging to that list are added

1. Make sure you have those Chicago episodes on your Plex server
2. Like the list on trakt.tv with your trakt account
3. Setup PTS with the same trakt account
4. make sure your config.yml enable liked_lists :
```
  trakt_to_plex:
    liked_lists: true
```
5. run a PTC sync

Only movies and episodes are added to playlist, read #531, #901 and first post of https://github.com/Taxel/PlexTraktSync/pull/76
